WalletWasabi.Gui/Global.cs

```diff
@@ -187,22 +191,7 @@ public async Task InitializeNoWalletAsync()
 				#region ProcessKillSubscription
 
 				AppDomain.CurrentDomain.ProcessExit += async (s, e) => await TryDesperateDequeueAllCoinsAsync();
-				Console.CancelKeyPress += async (s, e) =>
-				{
-					e.Cancel = true;
-					Logger.LogWarning("Process was signaled for killing.");
-
-					KillRequested = true;
-					await TryDesperateDequeueAllCoinsAsync();
-					Dispatcher.UIThread.PostLogException(() =>
-					{
-						var window = (Application.Current.ApplicationLifetime as IClassicDesktopStyleApplicationLifetime).MainWindow;
-						window?.Close();
-					});
-					await DisposeAsync();
-
-					Logger.LogSoftwareStopped("Wasabi");
-				};
+				Console.CancelKeyPress += async (s, e) => { e.Cancel = true; await StopAndExitAsync(); };
```

Comment:
I cannot see the diff here. I tried to go through it one by one but as far as I see nothing changed in the `StopAndExitAsync`.